unicorn/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+INCLUDE_DIRECTORIES(
+    ${PROJECT_SOURCE_DIR}/unicorn/include)
+
+ADD_LIBRARY(unicorn MODULE
+    include/cocaine/unicorn.hpp
+    include/cocaine/zookeeper/session
+    include/cocaine/zookeeper/connection
+    include/cocaine/zookeeper/handler
+    include/cocaine/zookeeper/exception
+    include/cocaine/unicorn/path
+    include/cocaine/unicorn/value
+    include/cocaine/unicorn/handlers
+
+    src/module
+    src/unicorn.cpp
+    src/unicorn/value
+    src/zookeeper/zookeeper
+    src/zookeeper/exception
+    src/zookeeper/session
+    src/zookeeper/handler
+    src/zookeeper/connection)
+
+TARGET_LINK_LIBRARIES(unicorn
+    msgpack
+    cocaine-core
+    zookeeper_mt
```

Comment:
From http://zookeeper.apache.org/doc/trunk/zookeeperProgrammers.html#C+Binding
"The package includes two shared libraries: zookeeper_st and zookeeper_mt. The former only provides the asynchronous APIs and callbacks for integrating into the application's event loop. The only reason this library exists is to support the platforms were a pthread library is not available or is unstable (i.e. FreeBSD 4.x). In all other cases, application developers should link with zookeeper_mt, as it includes support for both Sync and Async API."